### PR TITLE
fix: add bed notification

### LINKED ISSUE #8627: Modify bed creation notification to show proper action related notification
**Describe the bug**

When we are creating a single bed or multiple bed, all the time, the notification shown is " Bed(s) Created successfully "

**To Reproduce**

1. Open any facility card
2. Open manage facility and click on location management
3. Create a new location and click on the manage bed button
4. add a single bed only
5. see the error

**Expected behavior**

- [ ] If a single bed is created, show the notification: "Bed created successfully."
- [ ] If multiple beds are created, show the notification: "Beds created successfully."

**Screenshots**

image

### PR DESCRIPTION
**Pull request by AdityaJ2305**
## Fixes #8627 

- Added a ternary operator to check no. of beds added and display the notification as per no. of beds selected 

 ## video demo: 

https://github.com/user-attachments/assets/c57abfe9-cb79-4e1f-9059-1392fe8a6ae9

- Reviewers 
@ohcnetwork/care-fe-code-reviewers
@nihal467 

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Let's keep representable texts in `en` locale files using `useTranslations`.

**Review by rithviknishad — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
LGTM

File: src/Components/Facility/AddBedForm.tsx [hunk 1]
```diff
@@ -127,7 +127,10 @@ export const AddBedForm = ({ facilityId, locationId, bedId }: Props) => {
       const { res } = await request(routes.createFacilityBed, {
         body: { ...data, facility: facilityId, location: locationId },
       });
-      res?.ok && onSuccess("Bed(s) created successfully");
+      res?.ok &&
+        (numberOfBeds > 1
+          ? onSuccess(t("multiple_bed_created_notification"))
+          : onSuccess(t("single_bed_created_notification")));
```
Comment (bodhish) [inline src/Components/Facility/AddBedForm.tsx]: Look into https://www.i18next.com/translation-function/plurals

**Review by bodhish — CHANGES_REQUESTED** [review]
CC @rithviknishad

**Comment by khavinshankar** [comment]
@AdityaJ2305 can you make the necessary changes

**Comment by AdityaJ2305** [comment]
Yes by EOD

**Comment by AdityaJ2305** [comment]
@bodhish @rithviknishad   Should we display `4 Beds created successfully` (with the count) or keep the previous format, `Beds created successfully` ?

**Comment by rithviknishad** [comment]
Let's keep the count

**Review by bodhish — APPROVED** [review]
LGTM

**Pull request merged**